src/rpc/mining.cpp

```diff
@@ -458,9 +490,10 @@ UniValue getblocktemplate(const UniValue& params, bool fHelp)
         pindexPrev = pindexPrevNew;
     }
     CBlock* pblock = &pblocktemplate->block; // pointer for convenience
+    const Consensus::Params& cparams = Params().GetConsensus();
```

Comment:
Can you call this var consensusParams for consistency with other parts of the code?